### DISCUSSION TITLE
Which artemis crd version to use?

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#794] [Q&A] [answered]
Hi all,
I was looking at broker_activemqartemis_crd.yaml content and I saw that there are a list of versions supported by the operator from v1beta1 to v2alpha5.
Which one must be used to create an Artemis Broker? v2alpha5 is the greatest and latest?
How many versions will you support in the next future?
I would like to avoid using the oldest one that will not be supported in the near future.

**Comment by brusdev** [comment, marked as answer]
Hi @lordigon, the latest and greatest version is v1beta1 according with the [version priority in CustomResourceDefinitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#version-priority) and it is also the recommended version because it is used to store the CR.
